components/get_link_modal.tsx

```diff
@@ -10,6 +10,7 @@ import SuccessIcon from 'components/widgets/icons/fa_success_icon';
 type Props = {
     show: boolean;
     onHide: () => void;
```

Comment:
I think we should be able to remove the `onHide` prop entirely now that we have `onExited`

Reply:
Never mind. I see that this is a proper `onHide` method and not the one provided by `ModalController`.

Reply:
Acknowledged